exporter/coralogixexporter/client.go

```diff
@@ -70,13 +70,20 @@ func (c *coralogixClient) newPost(ctx context.Context, td pdata.Traces) error {
 	if err != nil {
 		return fmt.Errorf("can't translate to jaeger proto: %w", err)
 	}
-	ctx = metadata.NewOutgoingContext(ctx, metadata.New(c.cfg.GRPCClientSettings.Headers))
-	for _, batch := range batches {
 
+	for _, batch := range batches {
+		subsystem := batch.GetProcess().ServiceName
+		c.metadata = cxpb.Metadata{
+			ApplicationName: c.cfg.AppName,
+			SubsystemName:   subsystem,
+		}
+		c.cfg.GRPCClientSettings.Headers["subsystemName"] = subsystem
```

Comment:
If your user has set this header themselves in the config, you are overriding it without warning. Be nice to your users and inform them that this is happening.

Reply:
A warning message is added